src/clj/medusa/alert.clj

```diff
@@ -39,12 +39,42 @@
       (let [[earliest-build latest-build] (changesets/bounding-buildids date "mozilla-central") ; get the buildids for the given date
             changeset-url (changesets/pushlog-url earliest-build latest-build "mozilla-central")]
         (log/info "Changeset URL" changeset-url)
-        (send-email (str "Alert for " metric_name " (" detector_name ") on " date)
-                    (str "Alert details: " alert-url
+        (send-email (str "Telemetry Alert for " metric_name " on " date)
+                    (str "We have detected a change in the Telemetry probe " metric_name " in Firefox Nightly builds from " date
                          "\n\n"
-                         "Evolution dashboard: " evo-url
+                         "Alert details: " alert-url
+                         "\n"
+                         "A list of changes new to Nightly builds on " date ": " changeset-url
+                         "\n"
+                         "The value of " metric_name " over time: " evo-url
+                         "\n\n"
+                         "What to do about this:"
+                         "\n\n"
+                         "1. File a bug to track your investigation. You can just copy this email into the bug Description to get you started."
+                         "\n"
+                         "2. Reply-All to this email to let the list know that you are investigating. Include the bug number so we can help out."
+                         "\n"
+                         "3. Triage the alert. You can find instructions here: " alert-url
+                         "\n\n"
+                         "If you have any problems, please ask for help on the #telemetry IRC channel or on Slack in #fx-metrics. We'll give you a hand."
+                         "\n\n"
+                         "What is this:"
```

Comment:
I'd say "What this is:" or else "What is this?" I had to read it a few times before my brain quit stumbling over it. :-)